hydra-main/src/main/java/com/addthis/hydra/minion/Minion.java

```diff
@@ -262,16 +261,19 @@ private Minion(@JsonProperty("dataDir") File rootDir,
         }
         activeTaskKeys = new HashSet<>();
         jetty = new Server(webPort);
+
+        PrometheusServletCreator.create(handler);
         jetty.setHandler(minionHandler);
         jetty.start();
+        handler.start();
```

Comment:
Add comments on why handler.start/stop are required. Also I didn't mean move the code, please put it back where you think it should go (above)